### PR TITLE
Add control over page metadata

### PR DESCRIPTION
**Pull request by mattbowen-usds**
This adds

- Consistent metadata fields in frontmatter
- A component that's added to every page template for these fields.

### CODE REVIEW

**Comment by mattbowen-usds** [comment]
Note that I still need to document this, but Imma come back through with docs in the next sprint.

File: tech-far-hub/src/components/seo.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import { IMinimalFrontmatter } from "../types";
+
+export const SEO = ({ frontmatter }: { frontmatter?: IMinimalFrontmatter | null }) => {
```
Comment (vim-usds) [inline tech-far-hub/src/components/seo.tsx:4]: Haven't got my head around this new Head API

File: tech-far-hub/src/pages/template-listing.tsx [hunk 2]
```diff
@@ -55,7 +56,9 @@ const DefaultPageTemplate: React.FC<DefaultPageProps> = ({ data, children, pageC
 
 export default DefaultPageTemplate;
 
-export const Head: HeadFC = () => <title>TechFAR Hub</title>;
+export const Head: HeadFC<Queries.ListPageContentQuery> = ({ data }: { data: Queries.ListPageContentQuery }) => (
```
Comment (vim-usds) [inline tech-far-hub/src/pages/template-listing.tsx:59]: OK so each page template instantiates the SEO component using the Head API

**Review by vim-usds — APPROVED** [review]
Learned something new again!

**Pull request merged**